example/app.js

```diff
@@ -3,19 +3,11 @@ const { OM } = require('../');
 const apiKey = process.env.API_KEY;
 const apiSecret = process.env.API_SECRET;
 
-
-const om = new OM(apiKey, apiSecret, 'http://localhost:3200');
-
 (async () => {
 
-    await om.createContact({ phone: 123123123 }).catch(console.error);
-
-    setTimeout(async () => {
-        const contact = await om.createContact({ phone: '+18724000824', email: 'david@roncancio.me' }).catch(console.error);
-        console.log({ contact });
-    }, 2000);
+    const om = await OM(apiKey, apiSecret, 'https://next.omsg.io');
 
-    await om.createContact({ phone: '+18724000824', email: 'david@roncancio.me' }).catch(console.error);
+    console.log(await om.createContact({ phone: '+18724000824', email: 'david@roncancio.me' }).catch(console.error));
```

Comment:
I know the purpose is using this example as a way to test the functionality. If we can write a more verbose code in this special case to indicate how to use the SDK would be great.

I'm thinking about the scenario in which some external sources want to use it.


Maybe something similar to:
```js
try {
  return await om.createContact({ phone: '+18724000824', email: 'david@roncancio.me' });
} catch(error) {
  throw error;
}